src/dra/843B0.c

```diff
@@ -265,7 +265,222 @@ void func_80124A8C(Entity* entity) {
 INCLUDE_ASM("dra/nonmatchings/843B0", EntitySubwpnThrownDagger);
 
 // axe thrown when using subweapon
-INCLUDE_ASM("dra/nonmatchings/843B0", EntitySubwpnThrownAxe);
+// near-duplicate of EntitySubwpnCrashAgunea
+void EntitySubwpnAxeCrash(Entity* self) {
```

Comment:
I just checked and this is also called when Alucard uses the axe subweapon. Naming it Crash seems to imply it's only used there?